meta-balena-reference/qemux86-64.coffee

```diff
@@ -0,0 +1,51 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
```

Comment:
I'd rather see coffee files and svg files living outside meta-balena. If we move the submodules complexity to a balena-reference repo, maybe they could live there. Otherwise, maybe balena-yocto-scripts is a good place if we name it better.
I think coffee files will eventually disappear, so forward looking I think it's  best not to bring them over.